app/views/transactions/_form.html.slim

```diff
@@ -24,9 +24,8 @@
         = f.input :bank_account, label: false, disabled: true
     .col-md-3.col-sm-6.col-xs-6
       = f.input :comment, label: false
-    - if params[:controller] == 'transactions'
-      .col-md-3.col-sm-6.col-xs-6
-        = f.input :created_at, as: :string, label: false, input_html: { class: 'datepicker',
-          value: f.object.created_at.try(:strftime,'%d/%m/%Y') }
+    .col-md-3.col-sm-6.col-xs-6
+      = f.input :created_at, as: :string, label: false, input_html: { class: 'datepicker',
```

Comment:
@yunixon don't use `created_at` field for date. Create a new field `date` and use it.